### PR TITLE
close #11 home screen design

### LINKED ISSUE #11: Home Screen Design

### PR DESCRIPTION
**Pull request by darot-chen**

### CODE REVIEW

**Comment by darot-chen** [comment]
Screen Shot 2021-05-10 at 1 02 46 AM

**Comment by theachoem** [comment]
@darot-chen you forget about border & we use border-radius 4, not 10

image

File: lib/models/home_card_model.dart [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    this.onTap,
+  });
+
+  static getCards() => [
+        HomeCardModel(
+          title: 'Sender',
+          subTitle: 'Send string to arduino robot via Bluetooth',
+          icon: Icons.keyboard,
+          background: ThemeConstant.greenGradient,
+          isSmall: false,
```
Comment (theachoem) [inline lib/models/home_card_model.dart:27]: this should be `isBig`, since it will be a bigger card

**Comment by darot-chen** [comment]
Okok

File: lib/models/home_card_model.dart (hunk 1 above)
Comment (darot-chen) [reply lib/models/home_card_model.dart:27]: I think the default is small so I set it isSmall   😂

Comment (theachoem) [reply lib/models/home_card_model.dart:27]: Oh, I maybe confuse there, so that is okay

**Comment by theachoem** [comment]
@darot-chen in detail, icon in design also has shadow

**Pull request merged**